Makefile

```diff
@@ -0,0 +1,14 @@
+up:
+	docker-compose up -d
+
+down:
```

Comment:
Okay, I didn't know that this app would be an example for other devs. For me it would just be a demo to familiarize ourselves with the RGAPI.
I'll keep this in mind for the next feature we're coding.

I'll remove makefile